src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -133,15 +131,11 @@ public static String getHearingPriority(SscsCaseData caseData) {
         // Flag to Lauren - how  can this be captured in HMC queue?
         // If there's an adjournment - date shouldn't reset - should also go to top priority
```

Comment:
Comments shouldn't be needed, instead the code should speak for itself, otherwise it's too complicated.  That said, JavaDoc is encouraged for documentation (I know this project is generally missing JavaDoc)

Reply:
This isn't my comment, I assume its added by another ticket.